src/Send/MemoField.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import {defineMessages, useIntl} from 'react-intl'
+import {StyleSheet, Text, View} from 'react-native'
+
+import {TextInput} from '../components'
+import {COLORS} from '../theme'
+import {useSend} from './Context/SendContext'
+
+export const maxMemoLength = 256
+
+export const MemoField = () => {
+  const strings = useStrings()
+  const {memo, memoChanged} = useSend()
```

Comment:
can we receive the `memo` and `memoChange` as props? `{memo: string, onChange: (text: string) =>}`